src/leetcode_env/list.rs

```diff
@@ -79,6 +72,7 @@ impl From<Vec<i32>> for ListHead {
     }
 }
 
+// TODO: Test the happy path of going from a linked list to a vec
```

Comment:
TODO ?

Reply:
Yes it's a note for all the places I think need tests.